src/CSSTransitionGroup.js

```diff
@@ -53,7 +37,21 @@ class CSSTransitionGroup extends React.Component {
   }
 }
 
-CSSTransitionGroup.propTypes = propTypes;
-CSSTransitionGroup.defaultProps = defaultProps;
+CSSTransitionGroup.propTypes = {
```

Comment:
I don't really like the propTypes defined down here, is the plugin really not smart enough to extract them as they were?